components/clarinet-sdk/src-ts/vitest/index.ts

```diff
@@ -1,3 +1,4 @@
+import { bool } from "@stacks/transactions/dist/cl";
```

Comment:
Is this used?

Reply:
Ah, my bad, I removed it but in the follow up PR: https://github.com/hirosystems/clarinet/pull/1171/files#diff-c26bfb8a8ec5511c86cae0ab7e151b28db319eccdc06e2992fb679b4c5a8fb23L1

Marking as resolve for this one